scopus-import/src/main/java/no/sikt/nva/scopus/conversion/CristinConnection.java

```diff
@@ -57,30 +56,39 @@ public Organization getCristinOrganizationByCristinId(URI cristinOrgId) {
     }
 
     private String getBodyFromOrganizationResponse(HttpResponse<String> response) {
-        if (response.statusCode() != HttpURLConnection.HTTP_OK) {
+        logger.info("Organization response: {}", response.body());
```

Comment:
Tror vi betaler for logging. Så masse logging bør vi unngå. Kanskje du får ut fra responsen endepunktet vi forsøkte å kontakte og kun logge ut det?